src/nn/oe.rs

```diff
@@ -98,18 +138,32 @@ extern "C" {
     #[link_name = "\u{1}_ZN2nn2oe15SetCpuBoostModeENS0_12CpuBoostModeE"]
     pub fn SetCpuBoostMode(mode: CpuBoostMode);
 }
+
+pub fn set_cpu_boost_mode(mode: CpuBoostMode) {
+    unsafe {
+        SetCpuBoostMode(mode);
+    }
+}
+
 extern "C" {
     #[link_name = "_ZN2nn2oe14RestartProgramEPKvm"]
     pub fn RestartProgram(argv: *const u8, argc: u32) -> !;
 }
-pub fn RestartProgramNoArgs() -> ! {
+pub fn restart_program_no_args() -> ! {
```

Comment:
Check if skyline-rs uses this because I’m pretty sure that sounds a breaking change. If it does, maybe make a copy with the proper name and mark the other as deprecated (there is an attribute macro for that) and explain that it’ll be removed by the next major update